definitions/environments/dom/flow_v0.261.x-/dom.js

```diff
@@ -4622,3 +4083,121 @@ declare class Notification extends EventTarget {
 
   close(): void;
 }
+
+
+type HTMLElementMap = {
```

Comment:
This is also the approach taken by TS. Since we are already on this route, let's just align the name with TS as well. TS uses `HTMLElementTagNameMap`